src/core/src/core_logic/ExecutionConfig.py

```diff
@@ -69,15 +69,20 @@ def __init__(self, env_layer, composite_logger, execution_parameters):
 
         # Accommodation for bugs in higher-level components where 'Security' is being selected without selecting 'Critical' - should be rolled back no later than Jan 2022
         if self.included_classifications_list is not None and ('Security' in self.included_classifications_list and 'Critical' not in self.included_classifications_list):
-            self.composite_logger.log_debug("The included_classifications_list was corrected to include 'Critical' when 'Security' was specified.")
+            self.composite_logger.log_debug("The included_classifications_list was corrected to include 'Critical' "
+                                            "when 'Security' was specified.")
```

Comment:
Let's not do this -- we have more horizontal real-estate than vertical on our screens. We have a principle to avoid this in this code base except in very critical function calls.